api_config.py

```diff
@@ -4,9 +4,11 @@
 ALOOMA_USERNAME = os.environ.get('ALOOMA_USERNAME')
 ALOOMA_PASSWORD = os.environ.get('ALOOMA_PASSWORD')
 ALOOMA_ACCOUNT_NAME = os.environ.get('ALOOMA_ACCOUNT_NAME')
+REQUEST_TIMEOUT = os.environ.get('REQUEST_TIMEOUT', 300)
```

Comment:
This seems way too long for a request timeout to me. I would probably make it something like 10 seconds.

Reply:
hm. I made it 300 because the default the client library uses is 60 seconds, and that doesn't seem to be enough - we're timing out with that. I thought 300 might be reasonable because it looks like it does use that for some other types of requests.

Reply:
Hmm, seems crazy to me but you're right! Thanks for explaining 👍

Reply:
I guess another question is: if the request times out, does it still restream or not?